tlf.1.in

```diff
@@ -1656,6 +1656,10 @@ The station callsign used in messages; also used to determine the station's
 country, zone and continent.
 .
 .TP
+\fBOPERATION_MODE\fR=\fICQ\fR
+Default operation mode CQ for RUN and S&P for Search and Pounce
```

Comment:
wording suggestion:

Set operating mode for start up. Use \fICQ\fR for "run" and \fIS&P\fR for "search and pounce". The default mode is \fICQ\fR.